Sitefinity CLI/PackageManagement/NuGetApiClient.cs

```diff
@@ -249,6 +295,44 @@ private IList<NuGetPackage> ParseDependencies(string dependenciesString)
             return dependencies;
         }
 
+        private async Task<string> GetBaseAddress(HttpClient httpClient, string source)
```

Comment:
What is this base address?

Reply:
We fetch the base URL from the service index, as it is recommended, I have added a comment to clarify this a bit